src/modules/topos/tps_storage.c

```diff
@@ -1146,7 +1146,8 @@ int tps_db_load_branch(
 
 	if((get_cseq(msg)->method_id == METHOD_SUBSCRIBE)
 			|| ((get_cseq(msg)->method_id == METHOD_NOTIFY)
-					&& (msg->event && msg->event->len > 0))) {
+					&& (msg->event && msg->event->len > 0
+							&& strncmp(msg->event->body.s, "talk", 4) != 0))) {
```

Comment:
Yes, the previous version accepted any NOTIFY with an event, and therefore set `bInviteDlg = 0`. This doesn't work with the `Event: talk`, since in that specific case topos needs to look for an INVITE dialog instead of a SUBSCRIBE dialog. So it doesn't only accept event "talk", but the opposite: it accepts any event except "talk".